Feature/ds 252 - Updated documentation voice and tone, as well as added some uniformity. (#383)
gsoft-inc/wl-hopper: apps/docs/content/tokens/core/motion.mdx

### Easings

> [Easings](https://developer.mozilla.org/en-US/docs/Web/CSS/easing-function) denotes a mathematical function that describes the rate at which a numerical value changes. They adds life to motion, providing natural rests, they set the mood of an animation. Hopper provides 3 predefined easings, each having their use.

Reviewer: ```suggestion
[Easings](https://developer.mozilla.org/en-US/docs/Web/CSS/easing-function) denotes a mathematical function that describes the rate at which a numerical value changes.They add life to motion by providing natural rests and setting the mood of an animation. Hopper provides 3 predefined easings, each having their use.
```
